bridges/snowbridge/primitives/core/src/reward.rs

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2023 Snowfork <hello@snowfork.com>
+
+use sp_runtime::traits::Debug;
+use xcm::latest::{NetworkId, Location, Asset, Junction::GlobalConsensus};
+
+pub trait PaymentProcedure {
+	/// Error that may be returned by the procedure.
+	type Error: Debug;
+
+	/// Pay reward to the relayer from the account with provided params.
+	fn pay_reward(
+		relayer: Location,
+		reward: Asset,
+	) -> Result<(), Self::Error>;
+}
```

Comment:
Just curious is this trait finalized?  Do we need to add a `destination` param here?

For our use case it will be mint on AH but if the fee asset is in `DOT`, then maybe there is no need for a cross-chain call?